packages/sanity/src/desk/components/confirmDeleteDialog/ConfirmDeleteDialog.tsx

```diff
@@ -73,27 +74,21 @@ export function ConfirmDeleteDialog({
 
   return (
     <Dialog
-      width={1}
+      width={0}
```

Comment:
Is there a reason you are setting this to be `0`? I think it looks better with `width={1}`
![Screenshot 2023-11-01 at 08 14 00](https://github.com/sanity-io/sanity/assets/44635000/b5cb8bef-901b-44d5-bae9-814587e589c7)
![Screenshot 2023-11-01 at 08 13 52](https://github.com/sanity-io/sanity/assets/44635000/46db94da-aae7-47ce-a129-d2314421244b)